<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QAction{color:white;}
QMainWindow{color:white;background-color:black;}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextPlaybackButton">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Next Playback</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousPlaybackButton">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:white;}</string>
    </property>
    <property name="text">
     <string>Previous Playback</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentPlaybackLabel">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>70</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_Light">
    <property name="title">
     <string>Add Light</string>
    </property>
    <addaction name="addLightButton"/>
    <addaction name="addWholeRigButton"/>
   </widget>
   <widget class="QMenu" name="menuColour">
    <property name="title">
     <string>Colour</string>
    </property>
    <addaction name="selectColourButton"/>
    <addaction name="colourPickerButton"/>
    <addaction name="colourReplaceButton"/>
   </widget>
   <widget class="QMenu" name="menuSequence">
    <property name="title">
     <string>Sequence</string>
    </property>
    <addaction name="saveSequenceButton"/>
    <addaction name="openButton"/>
    <addaction name="autoSaveSequenceButton"/>
    <addaction name="editSequenceButton"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="savePlaybackButton"/>
   </widget>
   <addaction name="menuAdd_Light"/>
   <addaction name="menuColour"/>
   <addaction name="menuSequence"/>
   <addaction name="menuPlayback"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addLightButton">
   <property name="text">
    <string>Add Light</string>
   </property>
  </action>
  <action name="selectColourButton">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colour Mode</string>
   </property>
  </action>
  <action name="colourPickerButton">
   <property name="text">
    <string>Choose Colour</string>
   </property>
  </action>
  <action name="saveSequenceButton">
   <property name="text">
    <string>Save Sequence</string>
   </property>
  </action>
  <action name="openButton">
   <property name="text">
    <string>Open Sequence</string>
   </property>
  </action>
  <action name="autoSaveSequenceButton">
   <property name="text">
    <string>Auto-save sequence</string>
   </property>
  </action>
  <action name="savePlaybackButton">
   <property name="text">
    <string>Save Playback</string>
   </property>
  </action>
  <action name="colourReplaceButton">
   <property name="text">
    <string>Replace Colour</string>
   </property>
  </action>
  <action name="editSequenceButton">
   <property name="text">
    <string>Edit Sequence</string>
   </property>
  </action>
  <action name="addWholeRigButton">
   <property name="text">
    <string>Add whole rig</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
